### PR TITLE
[10.0][IMP] medical_pharmacy: Add concept of pharmacist

### PR DESCRIPTION
**Pull request by lasley**
* Add a pharmacist model and view for RPH concept

Depends:
- [x] #60 
- [x] #65

### CODE REVIEW

**Comment by lasley** [comment]
Well this kind of ballooned because of the issue with the primary view in the model. I tried to make the commits as best as possible. Damned lints got me at the end though!

It also adds in the fact that I forgot to add default pictures in the models. Turns out that's a thing too. I cherry picked the medical portion to my OCA/medical PR

File: medical_pharmacy/demo/medical_pharmacist_demo.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    </record>
+
+    <record id="res_partner_pharmacist_2" model="res.partner">
+        <field name="name">Brandon Feinstein</field>
+        <field name="city">Las Vegas</field>
+        <field name="state_id" ref="base.state_us_23" />
+        <field name="country_id" ref="base.us" />
+        <field name="zip">89119</field>
+        <field name="street">123 Example Way</field>
+        <field name="email">kthompson@example.com</field>
```
Comment (tedsalmon) [inline medical_pharmacy/demo/medical_pharmacist_demo.xml:33]: Out of scope but this email is a duplicate from the previous demo Rph.

File: medical_pharmacy/demo/medical_pharmacist_demo.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <field name="phone">+1 (702) 123-4567 x233</field>
+        <field name="parent_id" ref="base.main_partner" />
+        <field name="birthdate_date">1980-01-23</field>
+        <field name='gender'>female</field>
+        <field name="birthdate_date">1980-01-23</field>
```
Comment (tedsalmon) [inline medical_pharmacy/demo/medical_pharmacist_demo.xml:19]: Also out of scope but this is a duplicate of L17

**Review by tedsalmon — APPROVED** [review]
LGTM! Noticed some issues but they're out-of-scope and not breaking anything :)

**Pull request merged**